src/main/drivers/pwm_output.c

```diff
@@ -40,6 +40,11 @@ static pwmCompleteWriteFuncPtr pwmCompleteWritePtr = NULL;
 static pwmOutputPort_t servos[MAX_SUPPORTED_SERVOS];
 #endif
 
+#ifdef BEEPER_PWM
```

Comment:
As @AndersHoglund already commented, don't use `BEEPER_PWM`. Instead use `BEEPER`, and add `isPwm` to `beeperDevConfig_t`.